app/views/rubygems/_aside.html.erb

```diff
@@ -1,6 +1,7 @@
 <div class="gem__aside l-col--r--pad">
-  <% if @rubygem.linkset.present? && github_link = link_to_github(@rubygem) %>
-    <%= render partial: "rubygems/github_button", locals: { github_link: github_link } %>
+
+  <% if github_params = github_params(@rubygem) %>
```

Comment:
Does this "shadow" `github_params` variable?

Reply:
I am not sure what do you mean by shadow. I want to render following partial if `github_params(@rubygem)` doesn't return nil. used assignment to avoid calling same again in `locals`.

Reply:
`github_params` is method, but also becomes variable in here after assignment. This emits warning in pure Ruby if I understand well. Maybe not problem for erb.

Reply:
ah, yes. It does work for erb. changed var name anyway.